### ISSUE TITLE
False positive for `shorthand-property-no-redundant-values`

### ISSUE THREAD

**Issue by clintonbloodworth** [stylelint/stylelint#1372]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

The `shorthand-property-no-redundant-values` incorrectly warns when using the three-value syntax.

> Which rule, if any, is this issue related to?

`shorthand-property-no-redundant-values`

> What CSS is needed to reproduce this issue?

```css
.foo {
  margin: 1px 1px 1px;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "shorthand-property-no-redundant-values": true
  }
}
```

> Which version of stylelint are you using?

`6.5.1`.

> How are you running stylelint: CLI, PostCSS plugin, Node API?

[Sublime Linter plugin](https://packagecontrol.io/packages/SublimeLinter-contrib-stylelint).

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:
```shell
Shorthand: Shorthand form for property 'margin' should be written more concisely as '1px' instead of '1px 1px 1px'
```

**Comment by davidtheclark** [comment]
This is correct. You can shorten that declaration to `margin: 1px;` to accomplish the same thing.

I will close the issue because it seems that the rule is working as intended, but feel free to continue the discussion if you have questions or concerns..
